main/exercise/question.class.php

```diff
@@ -2302,13 +2497,13 @@ public static function get_count_course_medias($course_id)
     }
 
     /**
-     * @param int $course_id
+     * @param int $courseId
      *
      * @return array
      */
-    public static function prepare_course_media_select($course_id)
+    public static function prepare_course_media_select($courseId)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $media_list is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)